src/routes/(console)/project-[project]/functions/function-[function]/settings/updateRuntime.svelte

```diff
@@ -51,11 +67,12 @@
                 $func.providerRepositoryId || undefined,
                 $func.providerBranch || undefined,
                 $func.providerSilentMode || undefined,
-                $func.providerRootDirectory || undefined
+                $func.providerRootDirectory || undefined,
```

Comment:
We need PR for CLI as well, so CLI stores and sets this new attribute properly.

Reply:
Created the PR for CLI Here: https://github.com/appwrite/sdk-generator/pull/1025